File: shared/src/main/java/xyz/klinker/messenger/shared/shared_interfaces/IMessageListFragment.java
package xyz.klinker.messenger.shared.shared_interfaces;

public interface IMessageListFragment {

    long getConversationId();

    void setShouldPullDrafts(boolean pull);
    void loadMessages();
    void loadMessages(boolean addedNewMessage);
    void setDismissOnStartup();
    void setConversationUpdateInfo(String text);
}

File: shared/src/main/java/xyz/klinker/messenger/shared/shared_interfaces/IMessageListFragment.kt
package xyz.klinker.messenger.shared.shared_interfaces

interface IMessageListFragment {

    val conversationId: Long

    fun setShouldPullDrafts(pull: Boolean)
    fun loadMessages()
    fun loadMessages(addedNewMessage: Boolean)
    fun setDismissOnStartup()
    fun setConversationUpdateInfo(text: String)
}
